asreview/webapp/api.py

```diff
@@ -533,12 +535,49 @@ def api_get_prior(project_id):  # noqa: F401
                         lock_name="active",
                         project_id=project_id):
             label_history = read_label_history(project_id, subset=subset)
+            label_history.reverse()
+
+        # count labeled records and max pages
+        count = len(label_history)
+        max_page_calc = divmod(count, per_page)
+        if max_page_calc[1] == 0:
+            max_page = max_page_calc[0]
+        else:
+            max_page = max_page_calc[0] + 1
+
+        if page is not None:
+            # slice out records on specific page
+            if page <= max_page:
+                idx_start = page * per_page - per_page
+                idx_end = page * per_page
+                label_history = label_history[idx_start:idx_end]
+            else:
+                raise ValueError(f"Page {page - 1} is the last page.")
+
+            # set next & previous page
+            if page < max_page:
+                next_page = page + 1
+                if page > 1:
+                    previous_page = page - 1
+                else:
+                    previous_page = None
+            else:
+                next_page = None
+                previous_page = page - 1
+        else:
+            next_page = None
+            previous_page = None
 
```

Comment:
Nice. I think there are options to implement this in Flask or extensions as well. For now, this is fine.